commands/command_clean.go

```diff
@@ -77,8 +77,9 @@ func cleanCommand(cmd *cobra.Command, args []string) {
 func init() {
 	RegisterSubcommand(func() *cobra.Command {
 		return &cobra.Command{
-			Use: "clean",
-			Run: cleanCommand,
+			Use:    "clean",
+			Run:    cleanCommand,
+			PreRun: resolveLocalStorage,
```

Comment:
I would rather be explicit about which commands are using it up front. If it ends up being the majority, then perhaps it makes sense to add a helper function, but not now.